docs/im/ism/api.md

```diff
@@ -161,6 +161,10 @@ POST _opendistro/_ism/add/index_1
 }
 ```
 
+If you use a wildcard `*` while adding a policy to an index, the ISM plugin interprets `*` as all indices, including the `.opendistro-security` system index that stores user roles and tenants. A delete action in your policy might accidentally delete all user roles and tenants in your cluster.
+We recommend that you avoid using the broad `*` wildcard and instead add some prefix to it while specifying indices with the `_ism/add` API.
```

Comment:
Example here might help. "and instead add a prefix, such as `my-logs*`, when specifying indices"

Reply:
I'd go even further and make it a command. "Don't use the broad `*` wildcard, and instead add"